apps/designer/app/designer/features/style-panel/sections/typography/typography.tsx

```diff
@@ -1,19 +1,318 @@
 import type { RenderCategoryProps } from "../../style-sections";
-import { ShowMore } from "../../shared/show-more";
-import { renderProperty } from "../../style-sections";
+import { Flex, Grid, IconButton, Tooltip } from "@webstudio-is/design-system";
+import { PropertyName } from "../../shared/property-name";
+import {
+  ColorControl,
+  FontFamilyControl,
+  FontWeightControl,
+  SelectControl,
+  TextControl,
+} from "../../controls";
+import {
+  CrossSmallIcon,
+  EllipsesIcon,
+  TextDirectionLTRIcon,
+  TextDirectionRTLIcon,
+  TextAlignCenterIcon,
+  TextAlignJustifyIcon,
+  TextAlignLeftIcon,
+  TextAlignRightIcon,
+  TextCapitalizeIcon,
+  TextHyphenIcon,
+  TextItalicIcon,
+  TextLowercaseIcon,
+  TextStrikethroughIcon,
+  TextTruncateIcon,
+  TextUnderlineIcon,
+  TextUppercaseIcon,
+} from "@webstudio-is/icons";
+import { ToggleGroupControl } from "../../controls/toggle/toggle-control";
+import { ValuePickerPopover } from "../../shared/value-picker-popover";
 
-export const TypographySection = ({
-  styleConfigsByCategory,
-  moreStyleConfigsByCategory,
-}: RenderCategoryProps) => {
+export const TypographySection = (props: RenderCategoryProps) => {
   return (
-    <>
-      {styleConfigsByCategory.map((entry) => renderProperty(entry))}
-      <ShowMore
-        styleConfigs={moreStyleConfigsByCategory.map((entry) =>
-          renderProperty(entry)
-        )}
-      />
-    </>
+    <Flex css={{ gap: "$spacing$7" }} direction="column">
+      <TypographySectionFont {...props} />
+      <TypographySectionSizing {...props} />
+      <TypographySectionAdvanced {...props} />
+    </Flex>
   );
 };
+
+export const TypographySectionFont = (props: RenderCategoryProps) => {
+  const { sectionStyle } = props;
+
+  return (
+    <Grid
+      css={{
+        gap: "$spacing$5",
```

Comment:
use the gap prop

Reply:
before that it was actually mapped to the corresponding numbers, gap5 would be spacing$5, but how often this is the case with this many spacing values? idk

Reply:
we need to have a look at grids holistically and see if there is a set of values that makes sense, if not we can as well deprecate Grid